BUILD.bazel

```diff
@@ -368,7 +368,9 @@ cc_library(
         ":reference_errors",
         ":reference_index",
         ":reference_scope",
+        ":reference_stablehlovalue",
```

Comment:
Now that I'm looking at this name in the context of a PR, I'm realizing that it's genericity may be confusing in the scope of the entire project. Let's rename to `InterpreterValue`?